woocommerce/woocommerce
# [WIP] Use workflow dispatch that now works with GITHUB_TOKEN

### All Submissions:






### Changes proposed in this Pull Request:

As of September workflow dispatch can now be done with `GITHUB_TOKEN`. See [this blog post](https://github.blog/changelog/2022-09-08-github-actions-use-github_token-with-workflow_dispatch-and-repository_dispatch/) for more info.

This allows us to remove one of the few remaining custom tokens we were using to get around those limitations.








### How to test the changes in this Pull Request:

1) Fork this repository.
2) Enable issues in your fork from `https://github.com/YOURORG/woocommerce/settings`
3) Enable actions in your fork from `https://github.com/YOURORG/woocommerce/settings/actions`. Also, make sure workflows have read and write permissions and the ability to create and approve PRs.
4) Enable the scheduled workflow from `https://github.com/YOURORG/woocommerce/actions/workflows/release-code-freeze.yml`.
6) Merge this PR into `trunk` of your new fork.
7) In your fork, create a release `7.3.0`.
8) If there aren't already changes in `plugins/woocommerce/changelog`, in trunk, create several changes in `trunk` that add change files to the changelog dir (whether via PRs or just manually adding change files).
9) Run the action manually from `https://github.com/YOURORG/woocommerce/actions/workflows/release-code-freeze.yml` with "2022-07-14" as the value for time override. You may disable the slack ping or change the channel if you want.
10) The script should run, create `release/7.4` from trunk, create a `7.4.0` milestone, generate the changelog, create a PR against `trunk` that removes the change files, and a PR against `release/7.3` that updates the changelog and removes the change files.
11) **If you need to repeat testing, first delete the branches and the milestone created by this PR.**
12) **Remember, you've enabled the action, which should also run according to the cron schedule.** If you don't want that to happen, disable the action again before it runs. (It should be relatively harmless in your fork)


### FOR PR REVIEWER ONLY:
